platform/ios/MGLCategoryLoader.m

```diff
@@ -4,7 +4,16 @@
 #import "NSProcessInfo+MGLAdditions.h"
 #import "NSString+MGLAdditions.h"
 
+#import "MGLMapView.h"
+
 // https://github.com/mapbox/mapbox-gl-native/issues/2966
+// https://github.com/mapbox/mapbox-gl-native/issues/3113
+
+@interface MGLMapView (MGLCategoryLoader)
+
++ (void)mgl_linkMapView;
```

Comment:
Oh, good call. I was wondering about optimization there, and originally went with `+[UIView requiresConstraintBasedLayout]`, but something from `NSObject` would be safer.